geode-core/build.gradle

```diff
@@ -311,7 +313,7 @@ dependencies {
     implementation('com.healthmarketscience.rmiio:rmiio')
 
     //Geode-common has annotations and other pieces used geode-core
-    api(project(':geode-common'))
+    implementation(project(':geode-common'))
```

Comment:
This needs to be `api`. `geode-common` declares `Identifiable`, which is implemented by `CacheElement` in `geode-core`. There are other examples.

Reply:
Changed back to api.